models/Recipe.model.js

```diff
@@ -3,6 +3,55 @@ const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
   // TODO: write the schema
+  title: String,
```

Comment:
In the lab instruction they tell you thet the title needs to be unique and required:

```
  title: {
    type: String,
    required: true,
    unique: true,
  },
```